inc/include.php

```diff
@@ -12,7 +12,7 @@
 define('OMISE_MDL_CODE',              'mdl_omise');
 
 // Plugin version
-define('OMISE_PLUGIN_VERSION',        '2.2');
+define('OMISE_PLUGIN_VERSION',        '2.3');
```

Comment:
Please bump the version in a separate PR that also includes an update to the CHANGELOG e.g. https://github.com/omise/omise-eccube/pull/13

Reply:
@danfowler 
Sure, will do that.

Reply:
@danfowler 
Updated CHANGELOG.
https://github.com/omise/omise-eccube/pull/14/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

Reply:
@shunsuke-h-opn I was suggesting we make a separate PR to bump the version

Reply:
OK,created diffrent branch.

Reply:
@danfowler